src/performDocumentEdits.ts

```diff
@@ -0,0 +1,24 @@
+import { Edit } from "./types";
```

Comment:
```suggestion
import { Edit } from "./Types";
```

This breaks compile on case-sensitive file systems.